tst/testinstall/grppc.tst

```diff
@@ -1,16 +1,70 @@
 #@local F,G,S,c,cl,f,g,g1,g10,g3,gens,h,hh,i,m,n,pcgs,r,rws,sys,u,v,x,y
 gap> START_TEST("grppc.tst");
+gap> Display(TrivialGroup());
```

Comment:
This implicitly relies on `TrivialGroup` defaulting to create a pc group, which could change. So better to do this instead:
```suggestion
gap> Display(TrivialGroup(IsPcGroup));
```